pkg/logging/logging.go

```diff
@@ -61,7 +63,15 @@ func Logger() *zap.Logger {
 		cfg = zap.NewProductionConfig()
 	}
 
-	options := []zap.Option{zap.AddStacktrace(zapcore.FatalLevel)}
+	// Extract log fields from environment variables.
```

Comment:
What's the need for this? Won't filebeat/fluentd/<insert log shipper> do that for you?